src/hg_oap/units/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .dimension import *
+from .unit import *
+from .quantity import *
+from .unit_system import *
+from .U import U
```

Comment:
Given the implicit __enter__ in this code, would it be better to not import here?